source/qml/Library/RendererQml/AdaptiveCardQmlRenderer.cpp

```diff
@@ -1408,6 +1441,23 @@ namespace RendererQml
             uiContainer->Property("background", "Rectangle{border.width : 0; color: " + color + " }");
         }
 
+		int tempMargin = 0;
+
+		if (cardElement->GetPadding())
+		{
+			tempMargin = context->GetConfig()->GetSpacing().paddingSpacing;
+		}
+
+		uiColumn->Property("onHeightChanged", Formatter() << "{bodyLayout.generateStretchHeight(children, " << id << ".minHeight - " << 2 * tempMargin << " )}");
+
+		uiColumn->Property("onWidthChanged", Formatter() << "{bodyLayout.generateStretchHeight(children, " << id << ".minHeight - " << 2 * tempMargin << " )}");
+
+		uiContainer->Property("onMinHeightChanged", Formatter() << "{bodyLayout.generateStretchHeight( column_" << id << ".children, minHeight - " << 2 * tempMargin << " )}");
+
+		uiContainer->Property("property int minWidth", Formatter() << "bodyLayout.getMinWidth( column_" << cardElement->GetId() << ".children) + " << 2 * tempMargin);
```

Comment:
so, once we make the above change, here we can call `context->getCardRootId().generate......` instead of hardcoding as `bodyLayout`